phpstan.neon.dist

```diff
@@ -10,8 +10,6 @@ parameters:
     - tests/phpstan/stubs/wp-cli.php
     - tests/phpstan/stubs/exitexception.php
     - tests/phpstan/stubs/formatter.php
-    - vendor/php-stubs/wordpress-stubs/wordpress-stubs.php
```

Comment:
The stub are not needed?

Reply:
The stubs are loaded automatically by the phpstan-wordpress extension. Adding this line here is superfluous.